CONTRIBUTING.md

```diff
@@ -1,44 +1,65 @@
 Porting NCL example scripts to Python
 =====================================
-0. Fork and clone this repository and create a conda environment containing all of the software necessary to build this examples gallery webpage: [installation instructions](INSTALLATION.md)
 
-   Note that this repository uses a "git submodule" to store its data files in a separate GitHub repository. This submodule can be checked out as part of the clone process by using the `--recursive` flag:
-```
-git clone --recursive https://github.com/NCAR/GeoCAT-examples
-```
+1. Fork the [GeoCAT-examples GitHub repository](https://github.com/NCAR/GeoCAT-examples).
 
-   If you've already cloned the repository without the `--recursive` flag, you'll need to run the following command to clone the `data` submodule directory:
-```
-git submodule update --init
-```
+2. Clone your fork locally using [git](https://git-scm.com/), connect your repository to the upstream (main project), and create a new branch:
 
-   If you ever notice that a file exists in the [GeoCAT-datafiles](https://github.com/NCAR/geocat-datafiles) repository, but it does not seem to exist in your local `data` subdirectory, please try running the following command to sync the contents of the submodule:
-```
-git submodule update --remote
-git add data
-git commit -m "Update data submodule tracking to latest commit"
-```
+    ```bash
+    $ git clone git@github.com:YOUR_GITHUB_USERNAME/GeoCAT-examples.git
```

Comment:
Is there a preference for ssh over https? Just curious.

Reply:
No particular reason (it just happens to be my default). I switched to HTTPS since it is the easiest to set up

Reply:
Great. Thanks.